packages/story-editor/src/components/publishModal/constants.js

```diff
@@ -20,3 +20,6 @@ export const INPUT_KEYS = {
   SLUG: 'slug',
   TITLE: 'title',
 };
+
+export const HEADER_BAR_HEIGHT = '44px';
+export const HEADER_BAR_MARGIN = '1px';
```

Comment:
Ended up needing these to set height on the `Main` section of the modal to make scroll nice